packages/webapp/pages/posts/[id]/index.tsx

```diff
@@ -87,16 +110,10 @@ const PostPage = ({ id, initialData }: Props): ReactElement => {
     [PostType.Share, PostType.Welcome, PostType.Freeform].includes(post?.type),
     featureTheme && 'bg-transparent',
   );
-  const seoTitle = () => {
-    if (post?.type === PostType.Share && post?.title === null) {
-      return `Shared post at ${post?.source?.name}`;
-    }
```

Comment:
Moved the function outside the component so it isn't recreated every render, and follows best practices